tools/release.py

```diff
@@ -145,11 +145,8 @@ def topaddedversionstring(major, minor, rev):
     sonumber += 1
 
 for line in fileinput.input(cmakefile, inplace=1, backup='.bak'):
-    line = re.sub('SIMDJSON_SEMANTIC_VERSION "\d+\.\d+\.\d+','SIMDJSON_SEMANTIC_VERSION "'+newversionstring, line.rstrip())
+    line = re.sub('    VERSION \d+\.\d+\.\d+','    VERSION '+newmajorversionstring+'.'+mewminorversionstring+'.'+newrevversionstring, line)
```

Comment:
You need line.rstrip() on this line because the line that is read includes the line ending control characters. Latter we call 'print' which inserts new  line ending control characters, hence effectively inserting new blank lines between each line. It is either  line.rstrip() or we pass an 'end' parameter to the 'print' function below.

Reply:
Resolved in main branch.